gnosis/safe/addresses.py

```diff
@@ -1458,4 +1461,7 @@
             1675084,
         ),  # v1.3.0 default singleton address
     ],
+    EthereumNetwork.AURORIA_TESTNET: [
+        ("0x4e1DCf7AD4e460CfD30791CCC4F9c8a4f820ec67", 186758),  # v1.4.1+L2 
```

Comment:
```suggestion
        ("0x4e1DCf7AD4e460CfD30791CCC4F9c8a4f820ec67", 186758),  # v1.4.1
```
@falvaradorodriguez The proxy factory has not L2 version.